src/status_im/subs/wallet/collectibles.cljs

```diff
@@ -11,12 +11,24 @@
     image-url))
 
 (re-frame/reg-sub
- :wallet/collectibles
+ :wallet/collectibles-per-account
+ :<- [:wallet]
+ (fn [wallet [_ address]]
+   (as-> wallet $
+     (get-in $ [:accounts address :collectibles])
+     (map (fn [{:keys [collectible-data] :as collectible}]
+            (assoc collectible :preview-url (preview-url collectible-data)))
```

Comment:
@vkjr We are doing this transformation each time we display collectibles. WDYT about adding this key (`:preview-url`) when the collectibles are stored instead of each time we extract them?

I was about to do this change in this PR, but I'm not sure because subs are thought to do processing on the data that is useful to the views, so I'd say subs are the correct place for this processing, just a little uncomfortable to see this same piece of code for all subs